cli/submit.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import json
+import requests
+from getpass import getpass
+
+
+class StudySubmitter:
+    def __init__(self, ENA_AUTH_URL="https://www.ebi.ac.uk/ena/submit/webin/auth/token",
+                 WEBIN_SUBMIT_ENDPOINT ="http://www.ebi.ac.uk/eva/v1/submission/initiate/webin"):
+        self.ENA_AUTH_URL = ENA_AUTH_URL
+        self.WEBIN_SUBMIT_ENDPOINT = WEBIN_SUBMIT_ENDPOINT
+
+    # TODO
+    def submit_with_lsri_auth(self):
+        return NotImplementedError
+
+    # TODO
+    def upload_submission(self, submission_id, submission_upload_url):
+        pass
+
+    def _get_webin_credentials(self):
+        username = input("Enter your ENA Webin username: ")
+        password = getpass("Enter your ENA Webin password: ")
+        return username, password
+
+    def submit_with_webin_auth(self, username, password):
+        print("Proceeding with ENA Webin authentication...")
+
+        headers = {"accept": "*/*", "Content-Type": "application/json"}
+        data = {"authRealms": ["ENA"], "username": username, "password": password}
+        response = requests.post(self.ENA_AUTH_URL, headers=headers, data=json.dumps(data))
+
+        if response.status_code == 200:
+            print("Authentication successful")
```

Comment:
We've used logging everywhere else to provide information. I suggest we continue.

Reply:
Addressed